rubix-bookkeeper/src/main/java/com/qubole/rubix/bookkeeper/RetryingBookkeeperClient.java

```diff
@@ -97,21 +97,17 @@ public Void call()
         while (errors < maxRetries) {
             try {
                 if (!transport.isOpen()) {
-                    try {
-                        transport.open();
-                    }
-                    catch (Exception e1) {
-                        LOG.info("Error while reconnecting");
-                        continue;
-                    }
+                    transport.open();
```

Comment:
also, isnt this unncessary? The assumption says transport is already open and also when client is created, transport is opened explicitly as per assumption